src/app/products/product-shell/product-shell.component.ts

```diff
@@ -1,46 +1,9 @@
-import { Component, OnInit } from '@angular/core';
-import { Product } from '../product';
-import {
-    ProductStateFacadeService,
-} from '../state/product-state-facade.service';
+import { Component } from '@angular/core';
+import { ProductStoreService } from '../state/product.store.service';
 
 @Component({
     templateUrl: './product-shell.component.html',
 })
-export class ProductShellComponent implements OnInit {
```

Comment:
@mfp22 Thanks for the PR :)

The changes in the component code should be kept to a minimum.

All other refactors still have this code.

The main refactor should happen in de facade to make the comparison as fair as possible. That way the solutions can also be compared more easily.

Reply:
Methods are containers for imperative code, none of which should exist between component outputs and StateAdapt sources (or even just purely reactive RxJS code). However, for sake of comparison, I will add the methods back. But I consider the 100% reactive architecture recommendation an important aspect of StateAdapt.